libnetdata/log/log.c

```diff
@@ -897,6 +905,10 @@ void fatal_int( const char *file, const char *function, const unsigned long line
     snprintfz(action_result, 60, "%s:%s", program_name, strncmp(thread_tag, "STREAM_RECEIVER", strlen("STREAM_RECEIVER")) ? thread_tag : "[x]");
     send_statistics("FATAL", action_result, action_data);
 
+#ifdef NETDATA_INTERNAL_CHECKS
+    crash_netdata();
+#endif
+
```

Comment:
Hi, is this staying after the merge? A way to get more info via the core dump when it fatals? (Just asking in case it was used for testing or similar).

Reply:
I suggest to keep it. It only runs on `fatal()` when running with `NETDATA_INTERNAL_CHECKS`.

Reply:
It allows us to get a core dump to examine the situation of the fatal. Otherwise, it is not easy to understand how the fatal was triggered.